scripts/generate-upstream-patch.sh

```diff
@@ -2,46 +2,41 @@
 
 set -eo pipefail
 
-cleanup() {
-	rm -rf "$1"
-	echo "cleaned up $1"
-}
+ROOTDIR=$(git rev-parse --show-toplevel)
+UPSTREAM_URL_PREFIX="https://github.com/integritee-network"
 
-print_help() {
-	echo "Usage:"
-	echo "$0 [-h] [-p <tag|branch|commit-hash>] [-w <tag|branch|commit-hash>]"
-	echo ""
-	echo "without any parameter, the script will generate upstream patch for both pallets and tee-worker"
-	echo "	-h print this message"
-	echo "	-p specify the tag|branch|commit-hash for upstream pallets"
-	echo "	-w specify the tag|branch|commit-hash for upstream worker"
+function cleanup() {
+   rm -rf "$1"
+   echo "cleaned up $1"
 }
 
-check_upstream() {
-	local TARGET=$1
-	local UPSTREAM_URL="$UPSTREAM_URL_PREFIX/$TARGET"
-	if [ "$(git remote get-url upstream_$TARGET 2>/dev/null)" != "$UPSTREAM_URL" ]; then
-		git remote add upstream_$TARGET $UPSTREAM_URL
-	fi
+function usage() {
```

Comment:
It's not possible to generate patch for all at one time?
Normally `[ ]` represents optional parameters, `< >` represents madentary parameters.

Reply:
i think it's better to generate seperately.